auto_update_tests.py

```diff
@@ -226,4 +226,20 @@
   out = run_command(cmd, stderr=subprocess.STDOUT)
   open(os.path.join('test', 'binaryen.js', s + '.txt'), 'w').write(out)
 
+print '\n[ checking wasm-ctor-eval... ]\n'
+
+for t in os.listdir(os.path.join('test', 'ctor-eval')):
+  if t.endswith(('.wast', '.wasm')):
+    print '..', t
+    t = os.path.join('test', 'ctor-eval', t)
+    ctors = open(t + '.ctors').read().strip()
+    for opt in [0]:#, 1]:
```

Comment:
Should either have a `TODO` explaining why not opt=1, or ditch the `for` loop and `opt`-related flags